plugins/plugin-chart-echarts/src/Pie/controlPanel.tsx

```diff
@@ -52,17 +50,18 @@ const config: ControlPanelConfig = {
       label: t('Chart Options'),
       expanded: true,
       controlSetRows: [
-        ['color_scheme', noopControl],
+        ['color_scheme'],
         // eslint-disable-next-line react/jsx-key
         [<h1 className="section-header">{t('Legend')}</h1>],
         [showLegendControl],
-        [legendTypeControl, legendOrientationControl],
-        [legendMarginControl, noopControl],
+        [legendTypeControl],
+        [legendOrientationControl],
+        [legendMarginControl],
         // eslint-disable-next-line react/jsx-key
         [<h1 className="section-header">{t('Labels')}</h1>],
         [
           {
-            name: 'pie_label_type',
+            name: 'label_type',
```

Comment:
Does renaming this field matter here? Would you need a db migration for this?

Reply:
Yes, we need a migration - I'll add one to the bump PR.